extensions/amp-carousel/0.2/test/test-type-slides.js

```diff
@@ -62,6 +62,12 @@ function getSlideWrappers(el) {
   );
 }
 
+function isScreenReaderHidden(element) {
+ const computedStyle = getComputedStyle(element);
+ return  computedStyle.visibility === 'hidden' || 
+   computedStyle.display === 'none';
+}
```

Comment:
You may need to turn on format auto fix :) 
```suggestion
  return (
    computedStyle.visibility === 'hidden' || computedStyle.display === 'none'
  );
}
```